### PR TITLE
remove func GetStrategy()

### PR DESCRIPTION
**Pull request by soulseen**
Signed-off-by: soulseen <sunzhu@yunify.com>

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,4 +4,5 @@
 /.vagrant
 .vscode/
 cmd/s2i/debug
+.idea/
```
Comment (coreydaley) [inline .gitignore]: If you are using intellij products to do development, you should really put the .idea in your global gitignore file, as it's not a project specific file/folder that is being ignored.

https://help.github.com/en/articles/ignoring-files#create-a-global-gitignore

**Comment by coreydaley** [comment]
looks like you need to run `make verify` and fix any issues that it shows, probably with `gofmt -w pkg test `

File: .gitignore (hunk 1 above)
Comment (soulseen) [reply .gitignore]: ok

**Pull request closed without merging**